assets-yaml/xero_assets.yaml

```diff
@@ -2,187 +2,187 @@ openapi: 3.0.0
-                              }
-                            }
-                          },
-                          "items":[ 
-                            { 
-                              "assetId":"68f17094-af97-4f1b-b36b-013b45b6ad3c",
-                              "assetName":"Computer47822",
-                              "assetNumber":"123478074",
-                              "purchaseDate":"2020-01-01T00:00:00",
-                              "purchasePrice":100.0000,
-                              "disposalPrice":0.0,
-                              "assetStatus":"DRAFT",
-                              "trackingItems":[ 
-                            ],
-                            "bookDepreciationSetting":{ 
-                              "depreciableObjectId":"68f17094-af97-4f1b-b36b-013b45b6ad3c",
-                              "depreciableObjectType":"Asset",
-                              "bookEffectiveDateOfChangeId":"5da77739-7f22-4109-b0a0-67480fb89af0",
-                              "depreciationMethod":"StraightLine",
-                              "averagingMethod":"ActualDays",
-                              "depreciationRate":0.50,
-                              "depreciationCalculationMethod":"None"
-                            },
-                            "bookDepreciationDetail":{ 
-                              "depreciationStartDate":"2020-01-02T00:00:00",
-                              "priorAccumDepreciationAmount":0.000000,
-                              "currentAccumDepreciationAmount":0.000000,
-                              "currentCapitalGain":0.000000,
-                              "currentGainLoss":0.000000
-                            },
-                            "taxDepreciationSettings":[ 
-                            ],
-                            "taxDepreciationDetails":[ 
-                            ],
-                            "canRollback":true,
-                            "accountingBookValue":100.000000,
-                            "taxValues":[ 
-                            ],
-                            "isDeleteEnabledForDate":false
-                          },
-                          { 
-                            "assetId":"52ea3adf-f04a-4577-8fd2-43c52a256bd5",
-                            "assetName":"Computer4148",
-                            "assetNumber":"123466620",
-                            "purchaseDate":"2020-01-01T00:00:00",
-                            "purchasePrice":100.0000,
-                            "disposalPrice":0.0,
-                            "assetStatus":"DRAFT",
-                            "trackingItems":[ 
+                $ref: "#/components/schemas/Assets"
+              example: '{
+                "pagination":{
+                "page":1,
+                "pageSize":10,
+                "pageCount":2,
+                "itemCount":11,
```

Comment:
@jenksguo Love the YML linter.. but should all the indentation be removed for other examples?

Reply:
I've got concerns that single quotes where swapped out for double quotes.  This could cause issues when uploading the spec to getsandbox.  I think we may need to decline this PR but take the new components and add them directly to yaml without a tool that does formatting changes.